std-bits/base/src/main/java/org/enso/base/enso_cloud/EnsoSecretHelper.java

```diff
@@ -31,20 +31,20 @@ public final class EnsoSecretHelper extends SecretValueResolver {
   /**
    * Gets a JDBC connection resolving EnsoKeyValuePair into the properties.
    *
-   * @param properties properties in the form of {@code List<Pair<String, HideableValue>>}
+   * @param properties properties in the form of {@code HideableValue.KeyValuePair}
    */
   public static Connection getJDBCConnection(
-      String url, List<Pair<String, HideableValue>> properties) throws SQLException {
```

Comment:
- Removing usage of pair is good idea.
- Possibly we could remove `org/graalvm/sdk/collections/25.0.0/collections-25.0.0.jar` from classpath